qa/workunits/rbd/test_librbd_python.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -ex
 
+source $(dirname $0)/../ceph-helpers-root.sh
+install python3-nose
```

Comment:
Isn't this handled here [1]?

[1] https://github.com/ceph/ceph-cm-ansible/blob/master/roles/testnode/vars/redhat_8.yml#L50

Reply:
thanks, Jason. i am posting https://github.com/ceph/ceph-cm-ansible/pull/518 instead.